tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -478,6 +478,38 @@ def test_alice_expect_column_values_to_match_stftime_format_auto_yes_default_pro
     }
 
 
+@freeze_time(TIMESTAMP)
+def test_alice_expect_column_value_lengths_to_be_between_auto_yes_default_profiler_config_yes_custom_profiler_config_no(
+    alice_validator: Validator,
+) -> None:
+    validator: Validator = alice_validator
+
+    result: ExpectationValidationResult = (
+        validator.expect_column_value_lengths_to_be_between(
+            column="user_agent",
+            result_format="SUMMARY",
+            include_config=True,
+            auto=True,
+        )
+    )
+
+    assert result.success
+
+    expectation_config_kwargs: dict = result.expectation_config.kwargs
+    assert expectation_config_kwargs == {
+        "auto": True,
+        "batch_id": "cf28d8229c247275c8cc0f41b4ceb62d",
+        "column": "user_agent",
+        "include_config": True,
+        "max_value": 115,
```

Comment:
@cdkini It would be nice to also get an example where the min and the max values are different (and make sense to us).  Thanks!